packages/react/src/reconciler.ts

```diff
@@ -133,6 +139,12 @@ const reconciler = reactReconciler<
   scheduleDeferredCallback() {},
   cancelDeferredCallback() {},
 
+  // @ts-ignore -- @types/react-reconciler is several versions out of date
+  schedulePassiveEffects(fn:Function) {
```

Comment:
No CI setup but this should have a space after `fn:`

Reply:
doh